authors/apps/authentication/serializers.py

```diff
@@ -25,6 +30,51 @@ class Meta:
         # or response, including fields specified explicitly above.
         fields = ['email', 'username', 'password']
 
+    def validate_email(self, value):
+
+        email_regex = (r'^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$')
+        data = self.get_initial()
+        email = data.get("email")
+        user_query_set = User.objects.filter(email=email)
+
+        # validate that the correct email format is used
+        if not re.search(email_regex, email):
+            raise serializers.ValidationError(
+                "Incorrect email format please try again")
+        return value
+
+        if user_query_set.exists():
+            raise serializers.ValidationError(
+                "This email has already been used to create a user")
```

Comment:
hey @wasibani-roy, from a novice user`s perspective, I think ``` "This email has already been used to create a user"``` is confusing, i think it could be better when stated like  
*  "This email has already been used to create an account" or
* "This email is already in use" or 
*  "This email is taken"

Reply:
Changed this thanks